core/pom.xml

```diff
@@ -129,6 +129,12 @@
             <artifactId>evomaster-dbconstraint</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>com.graphql-java</groupId>
+            <artifactId>graphql-java</artifactId>
+            <version>13.0</version>
```

Comment:
the version declaration should be in the root pom file.
See MAVEN DEPENDENCY VERSION  in the for_developers.md docs

Reply:
Hi,

It was on my mind when I was writing that code but I was facing the following error when running graphql e2e test with graphql-java on root pom file:

https://paste.ubuntu.com/p/wH9JV5GmJW/

it was the reason that I added the following code in e2e-test pom file:

`            <exclusions>
                <exclusion>
                    <groupId>com.graphql-java</groupId>
                    <artifactId>graphql-java</artifactId>
                </exclusion>
            </exclusions>`

I excluded graphql-java from importing and I don't know any way to exclude a dependency from parent.

when I add graphql-java dependency to the e2e pom file it results this error and I spent a lot of time to figure it out but I got no answer for that !

Reply:
hi,
  it looks like you are having library version mismatches/conflicts.
Make sure to update the different GraphQL libraries to latest version:
eg, you have `graphql-java-tools` at 4.3.0, instead of 5.6.1 (but note the change in `groupId` with added `-kickstart`).
Same for `graphiql-spring-boot-starter`, where you have 4.0.0 instead of latest 5.10.0

Reply:
Hi,
the problem is solved now.